CHANGELOG.md

```diff
@@ -1,5 +1,106 @@
+- test: unit test `encryption_at_rest` resource [\#1750](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1750) ([oarbusi](https://github.com/oarbusi))
+- chore: Bump tj-actions/verify-changed-files from 7b7a3b8db9077729f56bd82fced85f4b0ee67bcd to 5eff60fda839b96c3e34d8239dffb116c900582c [\#1747](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1747) ([dependabot[bot]](https://github.com/apps/dependabot))
+- chore: Bump actions/stale from 8 to 9 [\#1746](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1746) ([dependabot[bot]](https://github.com/apps/dependabot))
+- chore: Bump actions/setup-go from 4 to 5 [\#1745](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1745) ([dependabot[bot]](https://github.com/apps/dependabot))
+- chore: Bump github.com/aws/aws-sdk-go from 1.48.13 to 1.49.1 [\#1744](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1744) ([dependabot[bot]](https://github.com/apps/dependabot))
+- chore: new scaffold command for creating resources/data sources [\#1739](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1739) ([AgustinBettati](https://github.com/AgustinBettati))
+- test: add unit test to `database_user`` resource [\#1738](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1738) ([oarbusi](https://github.com/oarbusi))
+- chore: lint error [\#1734](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1734) ([andreaangiolillo](https://github.com/andreaangiolillo))
+- doc: CLOUDP-215923: Removes sunset resources from documentation [\#1732](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1732) ([andreaangiolillo](https://github.com/andreaangiolillo))
+- chore: define atlas stream functionality under beta flag environment variable [\#1726](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1726) ([AgustinBettati](https://github.com/AgustinBettati))
+- chore: `migrate encryption_at_rest` resource to new SDK [\#1725](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1725) ([oarbusi](https://github.com/oarbusi))
+- chore: migrate database user resource to new SDK [\#1723](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1723) ([oarbusi](https://github.com/oarbusi))
+- doc: CLOUDP-216288 - Update the warning message in the cluster resource to mention a bug on the container ids [\#1719](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1719) ([andreaangiolillo](https://github.com/andreaangiolillo))
+- chore: Uses official CLI GH action in TF clean-up workflow [\#1718](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1718) ([lantoli](https://github.com/lantoli))
+- chore: Bump tj-actions/verify-changed-files from 1e75cac4ffa7ea5879addde1869f8fca09fce4c1 to 7b7a3b8db9077729f56bd82fced85f4b0ee67bcd [\#1713](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1713) ([dependabot[bot]](https://github.com/apps/dependabot))
```

Comment:
knitpit: I would put all bumps together at the end of the section

Reply:
adjusted